app/_how-tos/kic-get-started-proxy-caching.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Proxy Caching
+description: |
+  Cache all GET and HEAD responses across all Services using Proxy Cache and the KongClusterPlugin resource
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/get-started/proxy-caching/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Get Started
+
+series:
+  id: kic-get-started
+  position: 4
+
+tldr:
+  q: How do I cache upstream responses using {{ site.kic_product_name }}?
+  a: |
+    Use the `proxy-cache` plugin by creating a `KongPlugin` resource while specifying `config.response_code`, `config.request_method` and `config.cache_ttl`.
+
+products:
+  - kic
+
+tools:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+prereqs:
+  skip_product: true
+---
+
+## The Proxy Cache Plugin
+
+One of the ways Kong delivers performance is through caching. The [Proxy Cache plugin](/hub/kong-inc/proxy-cache/) accelerates performance by caching responses based on configurable response codes, content types, and request methods.  When caching is enabled, upstream services are not impacted by repetitive requests, because {{site.base_gateway}} responds on their behalf with cached results. Caching can be enabled on specific routes for all requests globally.
```

Comment:
```suggestion
One of the ways {{site.base_gateway}} delivers performance is through caching. The [Proxy Cache plugin](/plugins/proxy-cache/) accelerates performance by caching responses based on configurable response codes, content types, and request methods. When caching is enabled, upstream services are not impacted by repetitive requests, because {{site.base_gateway}} responds on their behalf with cached results. Caching can be enabled on specific Routes or for all requests globally.
```